### PR TITLE
begin patient admission form

### PR DESCRIPTION
**Pull request by tsognong**

### CODE REVIEW

File: src/components/accessories/admission/PatientAdmission.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import { IState } from "../../../types";
+import { AdmissionTransitionState } from "./types";
+import { initialFields } from "./consts";
+import { AdmissionDTO } from "../../../generated";
+
+const PatientAdmission: FC = () => {
+  const { t } = useTranslation();
+  const infoBoxRef = useRef<HTMLDivElement>(null);
+  const [shouldResetForm, setShouldResetForm] = useState(false);
+  const [shouldUpdateTable, setShouldUpdateTable] = useState(false);
```
Comment (axelpezzo) [inline src/components/accessories/admission/PatientAdmission.tsx:16]: Are we gonna use this in some place?

File: src/components/accessories/autocompleteField/AutocompleteField.tsx [hunk 2]
```diff
@@ -54,16 +54,15 @@ const AutocompleteField: FunctionComponent<IProps> = ({
...
             name={fieldName}
-            label={label}
             variant="outlined"
             error={isValid}
             fullWidth
           />
         )}
       />
-      <FormHelperText error>{errorText || ""}</FormHelperText>
     </FormControl>
```
Comment (axelpezzo) [inline src/components/accessories/autocompleteField/AutocompleteField.tsx:66]: We're not gonna use errorText?

File: src/components/accessories/discharge/PatientDischarge.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+import { AdmissionDTO } from "../../../generated";
+import { initialFields } from "./consts";
+import DischargeForm from "./dischargeForm/DischargeForm";
+import "./styles.scss";
+
+export const PatientDischarge: React.FC = () => {
+  const { t } = useTranslation();
+  const infoBoxRef = useRef<HTMLDivElement>(null);
+  const [shouldResetForm, setShouldResetForm] = useState(false);
+  const [shouldUpdateTable, setShouldUpdateTable] = useState(false);
```
Comment (axelpezzo) [inline src/components/accessories/discharge/PatientDischarge.tsx:12]: As before

Comment (tsognong) [reply src/components/accessories/discharge/PatientDischarge.tsx:12]: @axelpezzo I think, we could make simple, by closing the this PR #171 and the PR #173, so you can simply review the P #174

Comment (tsognong) [reply src/components/accessories/discharge/PatientDischarge.tsx:12]: Because I think All those works are used and corrected on PR #174

Comment (tsognong) [reply src/components/accessories/discharge/PatientDischarge.tsx:12]: please what do you think?

**Comment by tsognong** [comment]
This work is reused and corrected in PR #174 so you could just review that PR.

**Pull request closed without merging**